tools/jsii-compliance/suite.ts

```diff
@@ -18,612 +18,480 @@ export const suite: schema.Suite = {
       report: 'packages/@jsii/python-runtime-test/compliance-report.json',
     },
   },
-  exclusions: {
-    dotnet: {
-      reason:
-        'Underwent a different compliance process. Will be aligned in the future.',
-    },
-    python: {
-      reason:
-        'Underwent a different compliance process. Will be aligned in the future.',
-    },
-  },
   testCases: [
     {
       name: 'asyncOverrides_overrideCallsSuper',
       description: '',
-      exclusions: {},
     },
     {
       name: 'arrayReturnedByMethodCanBeRead',
       description:
         'Array created in the kernel can be queried for its elements',
-      exclusions: {},
     },
     {
       name: 'unionProperties',
       description:
         'Kernal values that accept a union type can be set, and are returned with the concrete type',
-      exclusions: {},
     },
     {
       name: 'syncOverrides',
       description: '',
-      exclusions: {},
     },
     {
       name: 'useEnumFromScopedModule',
       description:
         'Property that accepts an enum type from a third-party package can be set, and read',
-      exclusions: {},
     },
     {
       name: 'createObjectAndCtorOverloads',
       description:
         'Class can be instantiated with empty and non empty constructor',
-      exclusions: {
-        golang: {
-          reason:
-            'Golang does not have overloaded functions so the genearated class only has a single New function.',
```

Comment:
Should be migrated to the new mechanism.

Reply:
DONE